src/Common/src/Interop/Unix/libcurl/Interop.libcurl_multi.cs

```diff
@@ -46,16 +32,16 @@ internal static partial class libcurl
             SafeCurlHandle easy_handle);
 
         [DllImport(Interop.Libraries.LibCurl)]
-        public static extern int curl_multi_assign(
+        public static extern unsafe int curl_multi_wait(
             SafeCurlMultiHandle multi_handle,
-            curl_socket_t sockfd,
-            IntPtr sockptr);
+            curl_waitfd* extra_fds,
+            uint extra_nfds,
+            int timeout_ms,
+            out int numfds);
 
         [DllImport(Interop.Libraries.LibCurl)]
-        public static extern int curl_multi_socket_action(
+        public static extern int curl_multi_perform(
```

Comment:
Yes, I have a comment on that here: https://github.com/dotnet/corefx/pull/3050/files#diff-e5ffb568f6f994c4243d35ef1b6830ceR282